packages/driver/src/cypress.js

```diff
@@ -596,6 +599,21 @@ class $Cypress {
     return throwPrivateCommandInterface('addUtilityCommand')
   }
 
+  get currentTest () {
+    const r = this.cy.state('runnable')
+
+    if (!r) {
+      const invocationStack = $stackUtils.getInvocationDetails(this.specWindow, this.config)?.stack
```

Comment:
Not sure if this is a problem in practice, but in the docs we claim to support [nodejs v12](https://docs.cypress.io/guides/getting-started/installing-cypress#Node-js). But the `?` operator does not exist in nodejs v12. Would this cause a problem for a user running Cypress using nodejs 12?

Reply:
Users need Node.js 12+ to install Cypress. So all code within the `cli` needs to work in the lowest version we support for install. Other packages can use Node.js code from higher versions.

Reply:
Also, this is driver code that runs in the browser and not in Node.